test/libsolidity/syntaxTests/controlFlow/localStorageVariables/modifier_declaration_fine.sol

```diff
@@ -1,5 +1,5 @@
 contract C {
```

Comment:
I'm not quite sure what this test case is supposed to test actually... it doesn't have a "local storage variable" as suggested by the path... but well, that has little to do with this PR...

Reply:
I think it's from a time when modifiers behaved differently